src/wrappers/themis/react-native-themis/ios/RCTThemis.m

```diff
@@ -150,16 +140,14 @@ + (NSData*)dataDeserialize:(NSArray*) data
     callback(@[masterKey]);
 }
 
-- (TSCellSeal *)newSealMode: (NSArray*) symmetricKey
+- (TSCellSeal *)newSealMode: (NSArray*) symmetricKey error:(NSError**) error
 {
-    @try {
-        NSData *masterKey = [RCTThemis dataDeserialize: symmetricKey];
-        TSCellSeal *cell  = [[TSCellSeal alloc] initWithKey:masterKey];
-        return cell;
-    }
-    @catch (NSException *e) {
-        @throw e; // rethrow to catch in final function
+    NSData *masterKey = [RCTThemis dataDeserialize: symmetricKey error:error];
+    if (*error != nil) {
+        return nil;
     }
```

Comment:
Instead of checking the `error`, you should check the returned value.

`error` might be null if the caller does not care. It is not safe to dereference that pointer.

Of course, it is never null in the current version of the code, but [the pattern](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ErrorHandlingCocoa/CreateCustomizeNSError/CreateCustomizeNSError.html#//apple_ref/doc/uid/TP40001806-CH204-BAJIIGCC) is to check the returned value.

Reply:
fixed